include/Dialect/Btor/IR/BtorOps.td

```diff
@@ -859,9 +859,10 @@ def ConstantOp : Btor_Op<"constant", [ConstantLike, NoSideEffect,
   }];
 
   let arguments = (ins Builtin_IntegerAttr:$value);
-  let results = (outs AnyType:$result);
+  let results = (outs Btor_BitVec:$result);
 
-  let assemblyFormat = "attr-dict $value";
+  //let assemblyFormat = "attr-dict $value";
+  let assemblyFormat = "attr-dict `:` type($result)";
```

Comment:
This changes how the constant operation presents...can you not keep the attribute `value` displayed?